x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -1531,6 +1531,8 @@ public AsyncSender interceptSender(AsyncSender sender) {
         return templates -> {
             // .security index is not managed by using templates anymore
             templates.remove("security_audit_log");
+            // .security is a system index now. deleting another legacy template that's not used anymore
+            templates.remove("security-index-template");
```

Comment:
Security now uses a system index. FYI added this code to clean up an ancient typed index template @jkakavas